### PR TITLE
Refactor metric endpoint to static route

### PR DESCRIPTION
**Pull request by thomaschangsf**
Pull request refactor metric endpoint from dynamic path to static path without path /api/v1 offset.

### CODE REVIEW

File: acdc-ws/conf/routes [hunk 1]
```diff
@@ -1,3 +1,4 @@
 GET     /__status                   controllers.StatusController.status()
+GET     /__metrics                  controllers.MetricController.collect()
```
Comment (realstraw) [inline acdc-ws/conf/routes:2]: this would make the configuration of `acdc.metrics.endpoint` useless

Comment (realstraw) [reply acdc-ws/conf/routes:2]: I see, you removed it from application.conf too

Comment (thomaschangsf) [reply acdc-ws/conf/routes:2]: Yes, Simon and I talked and concluded a configurable metric endpoint would require introducing another router.  We chose a simpler approach to enable/disable metric collection via configuration.

**Review by damianxu88 — COMMENTED** [review]
LGTM

**Review by damianxu88 — APPROVED** [review]
LGTM

**Pull request merged**